tests/plugins/copyformatting/manual/undointegration.md

```diff
@@ -0,0 +1,20 @@
+@bender-ui: collapsed
+@bender-tags: bug, copyformatting, 4.11.3, 2780
+@bender-ckeditor-plugins: copyformatting, toolbar, wysiwygarea, undo, basicstyles
+
+## Test both editors
+1. Add some content to editor (e.g. new line), to "activate" undo step.
+2. Start to click around editor to change selection inside editor (20-25 times). Selection **have to differ** between adjacent steps.
+
+### Expected:
```

Comment:
Maybe add indentation to expectations? Thanks to that they'll become part of 2. step.